inference-engine/tests/functional/plugin/shared/include/single_layer_tests/detection_output.hpp

```diff
@@ -62,8 +62,8 @@ class DetectionOutputLayerTest : public testing::WithParamInterface<DetectionOut
     static std::string getTestCaseName(testing::TestParamInfo<DetectionOutputParams> obj);
     ngraph::op::DetectionOutputAttrs attrs;
     std::vector<InferenceEngine::SizeVector> inShapes;
-    void Infer() override;
-    void Compare(const std::vector<std::uint8_t> &expected, const InferenceEngine::Blob::Ptr &actual) override;
+    void infer() override;
+    void Compare(const std::vector<std::uint8_t> &expected, const InferenceEngine::Blob::Ptr &actual);
```

Comment:
minor comment, please use uniform style for the function name definition (`compare` instead of `Compare`)

Reply:
Done